### PR TITLE
Redirect user to original page after login

### PR DESCRIPTION
**Pull request by liaocanada**
Redirect user to original page after login

### CODE REVIEW

File: src/views/callback/LoginCallbackHandler.js [hunk 1]
```diff
@@ -27,7 +31,7 @@ const LoginCallBackHandler = () => {
     const username = accountData['cognito:username'];
     getUser(username).then(userAndOrders => dispatch(setUserData(userAndOrders.user)));
 
-    history.push('/?message=loginSuccess');
+    history.push(redirectPath || '/?message=loginSuccess');
```
Comment (phamEP) [inline src/views/callback/LoginCallbackHandler.js:34]: Im confused about this line. If this is an OR statement isnt that returning a boolean?

Comment (liaocanada) [reply src/views/callback/LoginCallbackHandler.js:34]: It's a cool JS thing. If redirectPath is truthy, the expression will evaluate to redirectPath. If it's falsy it'll evaluate to the second value

Comment (phamEP) [reply src/views/callback/LoginCallbackHandler.js:34]: Oh ok nice

Comment (liaocanada) [reply src/views/callback/LoginCallbackHandler.js:34]: Similar thing can be done with &&

`enabled && <h1>Feature</h1>`

If enabled is true it'll evaluate to the h1, if it's false then the whole thing becomes false so nothing is displayed

**Comment by liaocanada** [comment]
It's deployed now if you want to try

**Comment by phamEP** [comment]
It definitely works

**Comment by phamEP** [comment]
I accidently requested another review

**Pull request merged**